packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -265,5 +250,57 @@
     },
     "reloaded_with_new_preferences": "Reloaded with new preferences",
     "no_product_in_section": "There is no product in this section",
-    "what_are_you_looking_for": "What are you looking for?"
+    "what_are_you_looking_for": "What are you looking for?",
+
+    "plural_ago_minutes": "{count,plural, =0{Cached results from: less than a minute ago} =1{Cached results from: one minute ago} =2{Cached results from: two minutes ago} other{Cached results from: {count} minutes ago}}",
```

Comment:
It looks stupid to repeat `Cached results from` everywhere, doesn't it?
Regarding the separate version for `2` I don't see the point either: is that just a test for other languages with specific values for 2, 21 and so on?

Reply:
@yarons mentioned the same thing in #317, when we write it before the plural part it gets skipped. So we would have to put it in another translation and put them together in the code, like it was before. I left it in, but I dont know any language which could break it the clean way. 

Indeed it was for this, but it is only support for =0 =1 =2 =3